app/modules/scheduling/calendar_dates_router.py

```
@@ -0,0 +1,98 @@
+
+@router.post("", response_model=CalendarDateOut, status_code=201)
+def create_calendar_date(
+    payload: CalendarDateIn,
+    db: Session = Depends(get_db),
+    principal: Principal = Depends(require_role("admin", "scheduler")),
+):
+    row = m.TtCalendarDate(school_id=principal.school_id, **payload.model_dump())
+    db.add(row)
+    try:
+        db.commit()
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)